src/main/java/com/logdyn/ui/console/ConsoleApplication.java

```diff
@@ -52,10 +53,14 @@ private static void interactive() {
     }
 
     private static void execute(final String[] args) {
-        commands.stream()
-                .filter(command -> command.getName().equalsIgnoreCase(args[0]))
-                .findAny()
-                .orElse(HELP_COMMAND)
-                .execute(Arrays.copyOfRange(args, 1, args.length));
+        if (args.length > 1 && HELP_COMMAND.getNames().contains(args[1].toLowerCase())) {
```

Comment:
Maybe extract to boolean var to explain what it's checking for, I can't easily tell